### PR TITLE
create navigation to movies details

### PR DESCRIPTION
**Pull request by t-samborski**
The API downloading details has been slightly changed. A navigation has been created which, when clicked on a given film, takes you to its details.

### CODE REVIEW

File: src/features/MoviesBrowser/MovieDetails/Details/styled.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import styled from "styled-components";
 
 export const DetailsWrapper = styled.div`
-  max-width: 1368px;
+  max-width: 1920px;
```
Comment (KingaGrelewicz) [inline src/features/MoviesBrowser/MovieDetails/Details/styled.js:4]: why did you increse the width. Tile should be smaller than website.

File: src/features/MoviesBrowser/MovieDetails/Cast/index.js [hunk 2]
```diff
@@ -10,13 +10,13 @@ import { selectMovieCreditsSlice } from "./creditsSlice";
 
 const Cast = () => {
   const movieCredits = useSelector(selectMovieCreditsSlice);
-
+console.log(movieCredits)
   return (
     <>
       <CastHeader>Cast</CastHeader>
```
Comment (KingaGrelewicz) [inline src/features/MoviesBrowser/MovieDetails/Cast/index.js:16]: I changed Cast for Credits. Credits is overall component for cast and crew.

File: src/features/MoviesBrowser/PopularMovies/TileMovie/index.js [hunk 3]
```diff
@@ -8,21 +8,21 @@ import {
...
 
               <Container key={id}>
                   <Image src={`https://image.tmdb.org/t/p/original/${poster_path}`} alt="" />
-                  <Description>
-                      <DescriptionTitle>{title}</DescriptionTitle>
+                      <Description>
+                      <DescriptionTitle><Links to={`/movies/${id}`}>{title}</Links></DescriptionTitle>
                       <Data>{release_date}</Data>
                       <GenresList genre_ids={genre_ids}/>
                       <Rating>
```
Comment (KingaGrelewicz) [inline src/features/MoviesBrowser/PopularMovies/TileMovie/index.js:28]: here, You can use already created Rating component.
Rating has variants; you can add your variant if it's necessary,  we can use one component in a few places ;)

File: src/features/MoviesBrowser/MovieDetails/Details/styled.js (hunk 1 above)
Comment (t-samborski) [reply src/features/MoviesBrowser/MovieDetails/Details/styled.js:4]: Yes, I just wanted to check something with the image and I didn't change it to what it should be. An oversight :)

File: src/features/MoviesBrowser/MovieDetails/Cast/index.js (hunk 2 above)
Comment (t-samborski) [reply src/features/MoviesBrowser/MovieDetails/Cast/index.js:16]: There was also "Cast" in "main" when I downloaded it, so it copied the same to the branch. I hope there will be no major problems with merging.

File: src/features/MoviesBrowser/PopularMovies/TileMovie/index.js (hunk 3 above)
Comment (t-samborski) [reply src/features/MoviesBrowser/PopularMovies/TileMovie/index.js:28]: I thought about that too, but I'll deal with it in the next step.

File: src/features/MoviesBrowser/MovieDetails/Cast/index.js (hunk 2 above)
Comment (KingaGrelewicz) [reply src/features/MoviesBrowser/MovieDetails/Cast/index.js:16]: Yeah, Cast should be deleted ;)

File: src/features/MoviesBrowser/PopularMovies/TileMovie/index.js (hunk 3 above)
Comment (KingaGrelewicz) [reply src/features/MoviesBrowser/PopularMovies/TileMovie/index.js:28]: Great! Less means more 😅

**Review by KingaGrelewicz — APPROVED** [review]

**Pull request closed without merging**